### PR TITLE
Improve pulseaudio backend

### PR DESCRIPTION
**Pull request by pedrocr**
A few improvements for pulseaudio:

- Fix some uses of deprecated API
- Add a way to set a volume limit
- Expose mute/unmute in the CLI

### CODE REVIEW

File: src/audio/pulseaudio.rs [hunk 1]
```diff
@@ -362,10 +363,15 @@ impl PulseAudioSoundDevice {
             }
         };
 
+        // Since we're adjusting the volume unmute
+        self.set_muted(false)?;
```
Comment (kennylevinsen) [inline src/audio/pulseaudio.rs]: I do not think this is the correct thing to do: If the user wishes to unmute on change, they could simply call set_muted(false) themselves. With this, you cannot avoid it.

I personally adjust my volume before I unmute to not accidentally burst my eardrums.

**Review by kennylevinsen — CHANGES_REQUESTED** [review]
Thanks for the API upgrade. I do have one comment, but otherwise good.

File: src/audio/pulseaudio.rs (hunk 1 above)
Comment (pedrocr) [reply src/audio/pulseaudio.rs]: Yeah, that's the other option and pacmd does it like that. I'll need to expose the mute/unmute as well so I can chain two commands in one bindsym like I do with pacmd.

File: src/audio/pulseaudio.rs [hunk 2]
```diff
@@ -15,7 +14,9 @@ use libpulse_binding::context::{
 use libpulse_binding::mainloop::standard::IterateResult;
 use libpulse_binding::mainloop::standard::Mainloop;
 use libpulse_binding::proplist::{properties, Proplist};
-use libpulse_binding::volume::{ChannelVolumes, VOLUME_MAX, VOLUME_NORM};
+use libpulse_binding::volume::{ChannelVolumes, Volume, VOLUME_NORM};
+
+static MAX_VOLUME: Volume = Volume(131072); // 2^16 * 2 = 200%
```
Comment (pedrocr) [inline src/audio/pulseaudio.rs]: Any idea if there's a more adequate value? I was surprised to see pulse audio allow the volume to go up to 2000% and more, but now that I check pacmd doesn't have a limit either.

Comment (kennylevinsen) [reply src/audio/pulseaudio.rs]: Honestly, I think any limit we'd be setting is arbitrary, so I'm tempted to say that there shouldn't be a limit.

Comment (pedrocr) [reply src/audio/pulseaudio.rs]: Then maybe there should be an optional argument to add to set the cap. Something like:

```
platformctl audio -- +0.10 1.5
```

To increase volume by 10% up to 150% max.

Comment (kennylevinsen) [reply src/audio/pulseaudio.rs]: That sounds reasonable.

**Review by kennylevinsen — APPROVED** [review]
Awesome, thanks!

**Pull request merged**